M2/libraries/cddlib/Makefile.in

```diff
@@ -7,7 +7,7 @@ VERSION = 0.94m
 # cddlib is now available on github, so we could switch this library to a submodule
 # https://github.com/cddlib/cddlib
 # https://www.inf.ethz.ch/personal/fukudak/cdd_home/
-URL = http://macaulay2.com/Downloads/OtherSourceCode
+URL = https://macaulay2.com/Downloads/OtherSourceCode
```

Comment:
all of these standard URL variables are unnecessary, right?

Reply:
The autotools build uses it to fetch tarballs.

Reply:
Oh I thought the default one was included from Makefile.library.in. Then it wouldn't need to be repeated unless the url is different.

Reply:
Oh good call!  That definitely would work, but we're not currently doing it.  I'll push another commit with that change